src/provider/AbstractUnixProvider.php

```diff
@@ -9,6 +9,7 @@
 abstract class AbstractUnixProvider extends AbstractProvider
 {
     protected $cpuInfo;
+	protected $cpuInfoByLsCpu;
```

Comment:
good question :-) i think i forgot to move it after splitting into different classes. I'll move them both. Thank you!